### PR TITLE
WOR-1101 update rawls model version

### PR DESCRIPTION
**Pull request by blakery**
Updates the rawls model to pick up policies being returned on a workspace.

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -5,7 +5,7 @@ object Dependencies {
   val akkaHttpV = "10.2.10"
   val jacksonV = "2.13.4"
   val jacksonHotfixV = "2.13.4.2" // for when only some of the Jackson libs have hotfix releases
-  val nettyV = "4.1.94.Final"
+  val nettyV = "4.1.93.Final"
```
Comment (aherbst-broad) [inline project/Dependencies.scala]: Why this downgrade?

Comment (blakery) [reply project/Dependencies.scala]: rebase fail

**Comment by cahrens** [comment]
@blakery can you take a look at the swat failures?

File: automation/project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version = 1.9.1
+sbt.version = 1.9.2
```
Comment (blakery) [inline automation/project/build.properties]: I don't know why this is showing up as a change: it's the same version that was updated on the develop branch [here](https://github.com/broadinstitute/firecloud-orchestration/commit/01a9013557fd94a8d3d18bca51a6afcaa840cc64)

File: project/build.properties [hunk 3]
```diff
@@ -1 +1 @@
-sbt.version=1.9.1
+sbt.version=1.9.2
```
Comment (blakery) [inline project/build.properties]: as above, not sure why this is showing up in a diff

**Comment by blakery** [comment]
I'm closing this PR and opening a new one: whenever I merge/rebase this one it gets conflicts, and it should be ridiculously simple.

**Pull request closed without merging**